client/landing/stepper/declarative-flow/internals/steps-repository/migration-handler/index.tsx

```diff
@@ -65,17 +64,12 @@ const MigrationHandler: Step = function MigrationHandler( { navigation } ) {
 		return __( 'Scanning your site' );
 	};
 
-	const skipToDashboard = () => {
-		recordTracksEvent( 'calypso_importer_migration_skip_to_dashboard' );
-		return redirect( '/' );
-	};
-
 	const renderContent = () => {
 		if ( isUnAuthorized ) {
 			return (
 				<NotAuthorized
-					onStartBuilding={ skipToDashboard }
-					onStartBuildingText={ __( 'Skip to dashboard' ) }
+					type="source-site-not-connected-move-plugin"
+					startImport={ () => window.location.reload() }
```

Comment:
I think we'll need to pass in source site url in here

Reply:
Done!